src/app/components/plugin-uiframe/plugin-uiframe.component.html

```diff
@@ -10,4 +10,8 @@ <h3>Error</h3>
         <p>{{error.status}}</p>
         <button mat-stroked-button color="warn" (click)="error=null">dismiss</button>
     </div>
+    <button mat-icon-button *ngIf="isNisqPlugin" class="fsbutton mat-raised-button mat-primary" (click)="fullscreen=!fullscreen">
```

Comment:
the condition used here for fullscreen is rather specific. If possible make this more generic or leave a todo comment.